src/mango/src/mango_cursor_view.erl

```diff
@@ -124,25 +128,31 @@ composite_prefix([Col | Rest], Ranges) ->
             []
     end.
 
-
-% Low and behold our query planner. Or something.
-% So stupid, but we can fix this up later. First
-% pass: Sort the IndexRanges by (num_columns, idx_name)
-% and return the first element. Yes. Its going to
-% be that dumb for now.
+% Low and behold our query planner
```

Comment:
Why'd you remove ". Or something." Seems an odd change to make. Removing the snark is fine but I'd just remove the whole line. Also I think it should actually be "Lo and behold" but my Google is too far away to check for certain.